tests/unit/pages/Editor.spec.js

```diff
@@ -103,4 +121,867 @@ describe("Editor.vue", () => {
     expect(wrapper.vm.videoDBId).toEqual(dummyDraftPlio.data.video.id);
     expect(wrapper.vm.plioDBId).toEqual(dummyDraftPlio.data.id);
   });
+
+  it("saves plio in regular intervals if there's a change", async () => {
+    const savePlio = jest
+      .spyOn(Editor.methods, "savePlio")
+      .mockImplementation(() => {
+        return;
+      });
+    jest.spyOn(Editor.methods, "loadPlio").mockImplementation(() => {
+      return new Promise((resolve) => resolve());
+    });
+    const wrapper = mount(Editor);
+    const timeInterval = wrapper.vm.saveInterval;
+
+    // setInterval would've been called again after 5 seconds
+    // but as `changeInProgress` is false, `savePlio` will not be called
+    jest.advanceTimersByTime(timeInterval);
+    expect(savePlio).not.toHaveBeenCalled();
+
+    // change `changeInProgress` to true,
+    // and check before & after 5 seconds
+    await wrapper.setData({ changeInProgress: true });
+    expect(savePlio).not.toHaveBeenCalled();
+    jest.advanceTimersByTime(timeInterval);
+    expect(savePlio).toHaveBeenCalled();
+  });
+
+  it("saves plio when items are changed", async () => {
+    const checkAndSavePlio = jest.spyOn(Editor.methods, "checkAndSavePlio");
+    const wrapper = mount(Editor);
+
+    // items not changed, method not called at first
+    expect(checkAndSavePlio).not.toHaveBeenCalled();
+
+    // add items to the component, the method should've been called
+    await wrapper.setData({ items: dummyItemsCopy.data });
+    expect(checkAndSavePlio).toHaveBeenCalled();
+
+    // update the items, method should've been called
+    let updatedDummyItems = dummyItemsCopy.data;
+    updatedDummyItems[0].time = 20;
+    await wrapper.setData({ items: updatedDummyItems });
```

Comment:
this would work the same way right?
```suggestion
    dummyItemsCopy.data[0].time = 20;
    await wrapper.setData({ items: dummyItemsCopy.data });
```